common/winobjc.versioning.common.targets

```diff
@@ -6,6 +6,7 @@
     <UpdateAssemblyInfo>false</UpdateAssemblyInfo>
     <WriteVersionInfoToBuildLog>false</WriteVersionInfoToBuildLog>
     <_WinObjCVersioningTargetsImported>true</_WinObjCVersioningTargetsImported>
+    <GitVersionPath Condition="Exists('$(MSBuildThisFileDirectory)\..\.git')">$(MSBuildThisFileDirectory)\..\</GitVersionPath>
```

Comment:
It tells GitVersion where to look for the tags and other information. So when this guy is imported in a sample in another repo it would previously use that repo's tags which probably don't line up with ours. This roots it to the main repo as long as the .git directory is there (just being overly cautious).